cmd/brew-test-bot.rb

```diff
@@ -430,10 +430,11 @@ def download
         diff_end_sha1 = current_sha1
       # Use Azure Pipeline variables for master or branch jobs.
       elsif ENV["SYSTEM_PULLREQUEST_TARGETBRANCH"] && ENV["BUILD_SOURCEVERSION"]
+        system("git", "-C", @repository, "checkout", ENV["SYSTEM_PULLREQUEST_SOURCEBRANCH"])
```

Comment:
Can this turn into `test "git", "-C", @repository, "checkout", ....` up at line 416 to match something similar with Jenkins on line 399?

Reply:
(Once that's done I think this is good to merge)

Reply:
Done and tested! It works 👍